.github/workflows/build.yml

```diff
@@ -1,4 +1,4 @@
-name: Build
+name: build
```

Comment:
Humor me while I debug publishing. Will restore name with uppercase if it's not causing issues.